CMakeLists.txt

```diff
@@ -285,21 +286,21 @@ if (APPLE)
     set(ICON_OUTPUT "${CMAKE_BINARY_DIR}/AppIcon.icns")
     set(SOURCE_ICON_DIR "${CMAKE_CURRENT_SOURCE_DIR}/resources/logo")
 
-    add_custom_command(OUTPUT ${ICON_OUTPUT}
-        COMMAND cp icon_x16.png   ${ICON_TARGET}/icon_16x16.png
-        COMMAND cp icon_x32.png   ${ICON_TARGET}/icon_16x16@2x.png
-        COMMAND cp icon_x32.png   ${ICON_TARGET}/icon_32x32.png
-        COMMAND cp icon_x64.png   ${ICON_TARGET}/icon_32x32@2x.png
-        COMMAND cp icon_x128.png  ${ICON_TARGET}/icon_128x128.png
-        COMMAND cp icon_x256.png  ${ICON_TARGET}/icon_128x128@2x.png
-        COMMAND cp icon_x256.png  ${ICON_TARGET}/icon_256x256.png
-        COMMAND cp icon_x512.png  ${ICON_TARGET}/icon_256x256@2x.png
-        COMMAND cp icon_x512.png  ${ICON_TARGET}/icon_512x512.png
-        COMMAND cp icon_x1024.png ${ICON_TARGET}/icon_512x512@2x.png
-        COMMAND iconutil -c icns ${ICON_TARGET}
-        WORKING_DIRECTORY ${SOURCE_ICON_DIR})
-
-    set(BUNDLE_RESOURCES ${ICON_OUTPUT})
+#    add_custom_command(OUTPUT ${ICON_OUTPUT}
```

Comment:
Should this change be reverted to pack icons again?

Reply:
We just need to check if `iconutil` exists (it doesn't on osxcross), and then conditionally add it to the bundle resources.

Reply:
That means we don't automatically get icons for CI release builds, though, right? I'd really like to automate that as well. But I suppose we can tackle it in a follow-up PR.

Reply:
Correct. But I find not having icons less of an issue than not having any releases for the platform.

Reply:
Well, I have been building releases in a macOS VM so far… But it will be good to have this on the CI roster again to ensure we can actually build them.